modules/ROOT/nav.adoc

```diff
@@ -77,15 +77,15 @@ include::{partialsdir}/attributes.adoc[]
 
 
 
-** xref:security.adoc[Secure your app]
```

Comment:
changing the indentation suggests it's not a service.
revert to original indentation
but maybe change top level to 'Security'?